### DISCUSSION TITLE
Thank you for this

### DISCUSSION THREAD

**Discussion by tronikelis** [corvudev/corvu#4] [General]
This is exactly what solid is missing for further adoption of it. Thanks for this initiative 👍

**Comment by GiyoMoon** [comment]
Thank you a lot for this feedback! Really motivates me when reading kind messages like this 😇

**Comment by Lenstack** [comment]
Thanks for the hard work, 👍 'solidjs' really needs corvu. I'll be waiting for the stable version.

**Reply by tronikelis** [reply]
Lets go

**Comment by Clinton210** [comment]
Just started using this after trying out flowbite with solid. This worked a lot better and easier to use. Thanks so much!

**Comment by epatters** [comment]
Echoing these sentiments: thanks so much for this wonderful package! A much needed addition to the Solidjs ecosystem.
